tests/components/freedompro/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""Tests for the Freedompro integration."""
+from unittest.mock import patch
+
+from homeassistant.components.freedompro.const import DOMAIN
+
+from tests.common import MockConfigEntry
+from tests.components.freedompro.const import DEVICES, DEVICES_STATE
+
+
+async def init_integration(hass) -> MockConfigEntry:
```

Comment:
Should this be fixture instead?

Reply:
I use this feature to mock my plugin calls

Reply:
Yeah, I've seen, but it could be defined as a fixture in `conftest.py` instead.

See, for example here: <https://github.com/home-assistant/core/blob/dev/tests/components/ambee/conftest.py#L44>

And used here: <https://github.com/home-assistant/core/blob/dev/tests/components/ambee/test_sensor.py#L35>

Reply:
ok I think I understand